client/src/shared/components/SaleCard/SaleCard.tsx

```diff
@@ -1,13 +1,6 @@
 import style from './styles.module.scss';
+import { SaleCardProps } from '@/types/interfaces';
 import { FC } from 'react';
-
-interface SaleCardProps {
-  backgroundImage: string;
-  title: string;
-  subtitle: string;
-  discount: string;
-}
-
 export const SaleCard: FC<SaleCardProps> = (props) => {
```

Comment:
interface for the component can be placed above the component function

or placed in ./interfaces.ts in the same folder

(only shared interfaces are difined in shared interefaces file)